plugins/chat/config/locales/client.en.yml

```diff
@@ -693,8 +693,8 @@ en:
       titles:
         chat_mention: "Chat mention"
```

Comment:
Perhaps this should be "Mentioned in chat"

Reply:
good catch - done.